tests/testsuite/fix.rs

```diff
@@ -747,19 +834,6 @@ fn does_not_warn_about_dirty_ignored_files() {
     p.cargo("fix").run();
 }
 
-#[cargo_test]
-fn fix_all_targets_by_default() {
```

Comment:
Should we instead update this test for he new default behAVIOR?

Reply:
Yes, that sounds reasonable.